config/manager/kustomization.yaml

```diff
@@ -1,11 +1,11 @@
 resources:
-  - manager.yaml
+- manager.yaml
 
 generatorOptions:
   disableNameSuffixHash: true
-
-# Anton: not sure if the ConfigMap generated here by kubebuilder is used anywhere - so far let's omit it.
-#configMapGenerator:
-#  - name: manager-config
-#    files:
-#      - controller_manager_config.yaml
\ No newline at end of file
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/jianrzha/mongodb-atlas-operator
```

Comment:
can update here as well after migrate to team quay

Reply:
This is auto-generated :)